Makefile

```diff
@@ -8,7 +8,7 @@ install-node:
 	cd sandbox && npm install
 
 pre-commit-hook:
-	cp scripts/pre-commit .git/hooks/pre-commit
+	source .venv/bin/activate && pre-commit install
```

Comment:
Since `pre-commit` is already in poetry requirements, we can just activate the venv in the current sub-shell and run the `install` command to generate the local git hooks.